react/features/authentication/middleware.ts

```diff
@@ -289,7 +292,12 @@ function _handleLogin({ dispatch, getState }: IStore) {
                 titleKey: 'dialog.tokenAuthFailedTitle'
             }, NOTIFICATION_TIMEOUT_TYPE.LONG));
 
-            dispatch(redirectToDefaultLocation());
+            if (receivedTokenAuthServiceUrl) {
+                dispatch(openTokenAuthUrl(receivedTokenAuthServiceUrl,
+                    `${receivedTokenAuthServiceUrl}${receivedTokenAuthServiceUrl.includes('#')
+                        ? '&' : '#'}skipPrejoin=true`)
```

Comment:
Why do we need this? I think we need to handle this after we get the URL from with the token. But at that point we can use `appNavigate` which does have a `skipPrejoin` parameter.